test/e2e/scheduling/taints.go

```diff
@@ -283,7 +283,7 @@ var _ = SIGDescribe("NoExecuteTaintManager Single Pod [Serial]", func() {
 		Description: The Pod with toleration timeout scheduled on a tainted Node MUST not be
 		evicted if the taint is removed before toleration time ends.
 	*/
-	framework.ConformanceIt("removing taint cancels eviction", func() {
+	framework.ConformanceIt("removing taint cancels eviction [Disruptive]", func() {
```

Comment:
I am definitely not versed in how the scheduling/eviction works, but I'm just confused because your description doesn't seem to square with what is happening.

I didn't trust my understanding so I wanted to repro it myself. I built the kind images for the v1.16.0-rc.2 tag and ran just the `removing taint cancels eviction` test and reprod that the sonobuoy pod (a workload outside the test framework) gets evicted.

Here are some of the logs from the controller:
```
0913 18:54:07.405937       1 taint_manager.go:105] NoExecuteTaintManager is deleting Pod: heptio-sonobuoy/sonobuoy-e2e-job-3223075635be415d
I0913 18:54:07.406113       1 taint_manager.go:105] NoExecuteTaintManager is deleting Pod: heptio-sonobuoy/sonobuoy
I0913 18:54:07.406147       1 taint_manager.go:105] NoExecuteTaintManager is deleting Pod: kube-system/kindnet-lpx4w
I0913 18:54:07.406675       1 event.go:255] Event(v1.ObjectReference{Kind:"Pod", Namespace:"heptio-sonobuoy", Name:"sonobuoy", UID:"", APIVersion:"", ResourceVersion:"", FieldPath:""}): type: 'Normal' reason: 'TaintManagerEviction' Marking for deletion Pod heptio-sonobuoy/sonobuoy
I0913 18:54:07.406727       1 event.go:255] Event(v1.ObjectReference{Kind:"Pod", Namespace:"heptio-sonobuoy", Name:"sonobuoy-e2e-job-3223075635be415d", UID:"", APIVersion:"", ResourceVersion:"", FieldPath:""}): type: 'Normal' reason: 'TaintManagerEviction' Marking for deletion Pod heptio-sonobuoy/sonobuoy-e2e-job-3223075635be415d
```

Is this a bug? I see the eviction/deletion of the pod happen even before it logs that it is marking it for eviction.

Reply:
I think everyone here would welcome a PR if it made the tags more appropriate on the rest of the tests.  These two, being conformance related, have just a higher priority so I want to be sure to handle them for the release.